.drone.yml

```diff
@@ -3,7 +3,7 @@ name: default
 
 steps:
   - name: setup-minio
-    image: minio/mc
+    image: minio/mc:RELEASE.2019-07-31T19-17-35Z
```

Comment:
Is this change meant for this pull request?

Reply:
Yes, temporary. Current `latest` version of this image has issues so I am using the latest one that works.